wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -237,16 +242,16 @@ def populate_container_resources(
             )
 
     def wait_job_launch(
-        self, job_name: str, namespace: str, core_api: "CoreV1Api"
+        self, job_name: str, namespace: str, core_api: CoreV1Api  # type: ignore
     ) -> List[str]:
-        pods = core_api.list_namespaced_pod(
+        pods = core_api.list_namespaced_pod(  # type: ignore[attr-defined]
```

Comment:
It seems like with the types here we shouldn't need these ignores?

Reply:
Removing this still causes `error: CoreV1Api? has no attribute "read_namespaced_job_status"  [attr-defined]` :/